pupil-spa/src/app/services/speech/speech.service.ts

```diff
@@ -147,13 +147,11 @@ export class SpeechService implements OnDestroy {
   }
 
   /**
-   * Parse the source of a NativeElement and speak the text
+   * Removes aria-hidden child elements
    * @param nativeElement
+   * @returns {clonedElement}
    */
-  speakElement(nativeElement): Promise<{}> {
-    this.focusInterruptedPageSpeech = false;
-    const elementsToSpeak = 'h1, h2, h3, h4, h5, h6, p, li, button, a, span, fieldset';
-
+  removeUnspokenElements(nativeElement): HTMLElement {
```

Comment:
@danwilliams I see the `speakFocusedElement` already clones the element.
Does this function still need to clone the already cloned node?